st2common/st2common/config.py

```diff
@@ -179,16 +179,24 @@ def register_opts(ignore_errors=False):
 
     do_register_opts(webui_opts, "webui", ignore_errors)
 
+    # NOTE: These options, where applicable, try to use the standard mongo name, even though
```

Comment:
Are we saying the reason for camel case in `st2.conf` is so debug presents the same name as the mongo documentation?

Reply:
Yes, though searching for `tlsFoo` vs `tls_foo` is probably not a big deal. So, I'm flexible here.

Pymongo changed their arg names to match a standard for mongo libraries across all languages:
- https://pymongo.readthedocs.io/en/stable/migrate-to-pymongo4.html#renamed-uri-options
- https://github.com/mongodb/specifications/blob/master/source/uri-options/uri-options.md

I debated whether to follow the same standard or use snake_case. I decided to try following it and see what feedback TSC members give. Switching option names from camelCase to snake_case should be fairly simple since I kept snake_case within the code.

Does anyone else have a preference here?